pkg/server/events.go

```diff
@@ -98,6 +99,7 @@ func (em *eventMonitor) subscribe(subscriber events.Subscriber) {
 		`topic=="/tasks/exit"`,
 		`topic=="/tasks/oom"`,
 		`topic~="/images/"`,
+		`namespace==` + constants.K8sContainerdNamespace,
```

Comment:
Add double quote for the namespace for consistency.

Actually I can't find a document for this immediately, let's keep it the same with the previous code to make sure it works.

fmt.Sprintf(`namespace=="%s"`, constants.K8sContainerdNamespace)

Reply:
done

Reply:
had to switch to manual filter on our side..